crowdin/messages.json

```diff
@@ -129,13 +129,15 @@
   "115911783": "Of course, this list isn’t complete. Every day, impersonators come up with new ways to try to steal your information and money.",
   "117174749": "Can I close an open position before expiry?",
   "117273069": "<0>Wall Street 30</0> follows the stock performance of the top 30 listed companies in the US.",
+  "117318539": "Password should have lower and uppercase English letters with numbers.",
```

Comment:
Are we having changes on strings that need to be translated? are we handling it in this PR? Cause seems you didn’t pull any translation 🙏

Reply:
Yes, correct, didn't pull. First part - development, second - design and content. So I have to waiting for correct text